### DISCUSSION TITLE
Launch of Pybullet

### DISCUSSION THREAD

**Discussion by ze94** [WBK-Robotics/pybullet_industrial#49] [General]
Hi together, 

I just started my master thesis which will consist on developing an energy efficient physical simulation of an industrial robot in Pybullet. 
I installed pybullet with **pip install pybullet_industrial** and the corresponding build tools. 
Now being an absolute beginner in programming, I don't know how to properly launch pybullet. 
Will it be through Visual Code, Pycharm or any other terminal. 
Any type of help is welcomed. 
Thank you

**Comment by liquidcronos** [comment]
Hey,

this package deals with pybullet industrial which tries to expand pybullet to handle industrial robotics usecases.
For a thorough tutorial on pybullet please check pybullets documentation:
https://pybullet.org/wordpress/index.php/forum-2/

That said, the tutorial section shows code that will start a simulation:
https://pybullet-industrial.readthedocs.io/en/latest/tutorials.html
